### PR TITLE
feat: use BaseButton from react-native-gesture-handler

### PR DESCRIPTION
**Pull request by lukewalczak**
### Motivation

Solving the issue no #500.

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -1,4 +1,8 @@
 {
+  "globals": {
```
Comment (satya164) [inline .eslintrc]: We should remove this

File: src/components/TouchableRipple.js [hunk 2]
```diff
@@ -145,6 +123,19 @@ class TouchableRipple extends React.Component<Props, void> {
...
         {React.Children.only(children)}
       </TouchableHighlight>
+    ) : (
+      <BaseButton
+        {...rest}
+        disabled={disabled}
+        style={[style]}
+        rippleColor={
+          background !== undefined
+            ? background
```
Comment (satya164) [inline src/components/TouchableRipple.js]: We should get rid of `background` prop

**Comment by ferrannp** [comment]
We need to test if it works with https://github.com/kmagiera/react-native-gesture-handler/pull/400 cc @Trancever

**Comment by ferrannp** [comment]
@satya164 gesture handle is out, 3.0-alpha time.

**Comment by Trancever** [comment]
@ferrannp We wait for new expo release with updated gesture handler, without it we can't test it in our example app.

**Comment by ferrannp** [comment]
@brentvatne new Expo release with Gesture Handler coming soon? ❤️

**Comment by ferrannp** [comment]
We won't have this because the buttons are not accessible yet https://github.com/kmagiera/react-native-gesture-handler/issues/468 :(

**Comment by waqas19921** [comment]
@ferrannp There is fix for that now.

**Comment by RichardLindhout** [comment]
There is now also web support.

**Pull request closed without merging**